### PR TITLE
Remove AMI from public docs

### PR DESCRIPTION
**Pull request by glasner**
Took a first crack at removing all references to the AMI from our public docs (including sidebar). @evanphx going to add a few questions below. 

Unrelated: Do we have a preferred style for PRs? I just squashed all my commits, but happy to break back apart if easier.

### CODE REVIEW

File: content/source/docs/enterprise/private/diagnostics.html.md [hunk 1]
```diff
@@ -23,100 +23,12 @@ On the next page, click the _Download Support Bundle_ button which will download
...
-    tar -czf hashicorp-support.tar.gz *
-    gpg2 -e -r "Terraform Enterprise Support" \
-        --cipher-algo AES256 \
-        --compress-algo ZLIB \
-        -o hashicorp-support.tar.gz.enc \
-        hashicorp-support.tar.gz
-
-You will note that we first create a support bundle using the normal procedure,
-extract it, remove the files we want to omit, and then create a new one.
-
```
Comment (glasner) [inline content/source/docs/enterprise/private/diagnostics.html.md]: Do we need to make instructions for how to do this on the installer versions?

File: content/source/docs/enterprise/private/faq.html.md [hunk 2]
```diff
@@ -167,105 +148,6 @@ Additionally these hostnames are accessed by the Installer product in online mod
...
-
-The Private Terraform Enterprise AMI is based on the latest release of Ubuntu 16.04 with all security
-patches applied.
-
-### Network Ports
-
-The Private Terraform Enterprise AMI requires that port :8080 be accessible. This is where all traffic
-from the ELB is routed. Many other internal Private Terraform Enterprise services listen on the host,
-but they do not require external traffic. The AWS security group for the
-instance as well as software firewall rules within the runtime enforce this.
```
Comment (glasner) [inline content/source/docs/enterprise/private/faq.html.md]: Should this be rewritten to be platform agnostic and kept?

Comment (glasner) [reply content/source/docs/enterprise/private/faq.html.md]: I guess that goes for anything else below on the FAQ

File: content/source/docs/enterprise/private/faq.html.md [hunk 3]
```diff
@@ -310,7 +192,7 @@ Now, if you keep the `backend.tf` file in scope when you run `terraform` operati
 
 ## Private Terraform Enterprise Architecture
 
-This document describes aspects of the architecture of Private Terraform Enterprise as deployed via the AMI.
+This document describes aspects of the architecture of Private Terraform Enterprise.
```
Comment (glasner) [inline content/source/docs/enterprise/private/faq.html.md]: As I understand it, this diagram is a little outdated by still applies to installer version https://www.terraform.io/docs/enterprise/private/faq.html#private-terraform-enterprise-architecture

**Comment by nfagerlund** [comment]
I think I agree with Alexis. 

Standards for PRs in this repo are a little fast and loose. 😅 Squashing is fine!

**Comment by glasner** [comment]
@nfagerlund is it cool to keep this PR around for later? Do I need to mark it as such somehow?

**Comment by nfagerlund** [comment]
@glasner Yeah, sure — I usually change the title to "DON'T MERGE: (actual title goes here)" and then change it back before merging. Sometimes I leave an explanation of what I'm waiting for in the description, so people can tell at a glance whether I forgot something and they need to come bug me for it. :)

**Comment by nfagerlund** [comment]
I'm leaving this PR frozen for now. It'll have to be reconstructed as we rearrange the ptfe docs in the cloud / enterprise split.

**Comment by nfagerlund** [comment]
I've reconstructed these changes in https://github.com/hashicorp/terraform-website/tree/july19_cloud_enterprise_divide, which, PR incoming soonish.

**Pull request closed without merging**